classes/import-export.php

```diff
@@ -298,7 +298,7 @@ public function import_csv() {
 				// Include CSV library.
 				require_once( SIMPLEMAP_PATH . '/classes/parsecsv.lib.php' );
 
-				$file_location = WP_PLUGIN_DIR . '/sm-temp-csv-' . $blog_id . '.csv';
+				$file_location = $this->get_import_path( $blog_id . '.csv' );
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->